tsdef/LocalTrackOptions.d.ts

```diff
@@ -0,0 +1,7 @@
+import { LogLevel } from './Members';
```

Comment:
why `Members` and not `loglevel` ?

Reply:
Because the Members contains `AudioCodec, LogLevel, VideoCodec` in them as they're reused in several places, DefinitelyTyped also had it named Members. However, rename this folder to something else that encompasses these three if there is a more preferred name for these. Please let me know :)